index.js

```diff
@@ -3,6 +3,7 @@ const commander = require('commander')
 
 const scrapeKlerosTokenActivation = require('./src/scrapers/kleros-token-activation')
 const scrapeArbitrablePermissionList = require('./src/scrapers/arbitrable-permission-list')
+const scrapeKlerosAthenaSetStake = require('./src/scrapers/kleros-athena-set-stake')
```

Comment:
`scrapeKlerosLiquidSetStake` `kleros-liquid-set-stake`

We don't use the word Athena anywhere in code.